methodology/README.md

```diff
@@ -1 +1,56 @@
+
+## Methodologies and Standards
+
+Understanding and implementing major documented methodologies and standards are crucial elements of our approach. This empowers us to formulate strategies that leverage established practices, enhancing accountability and defensibility in our results.
+
+### Testing Environments
+
+Penetration testing methodologies often categorize tests based on the level of information provided to the tester. Common testing environments include:
+
+**1. Unknown-Environment Test:**
+   - Limited information provided (e.g., domain names and IP addresses).
+   - Mimics an external attacker's perspective, starting with minimal knowledge.
+   - Enhances realism by withholding information from network support personnel.
+
+**2. Known-Environment Test:**
+   - Tester possesses significant information about the organization and its infrastructure.
+   - Includes network diagrams, IP addresses, configurations, user credentials, and source code if applicable.
+   - Aims to identify security vulnerabilities within a broader scope.
+
+**3. Partially Known Environment Test:**
+   - A hybrid approach between unknown- and known-environment tests.
+   - Testers may be provided credentials but not full documentation of the network infrastructure.
+   - Allows testing from an external attacker's perspective while retaining some internal insights.
+
+### Common Penetration Testing Methodologies
+
+Several widely recognized methodologies and standards guide our penetration testing efforts:
+
+**1. [MITRE ATT&CK Framework](https://attack.mitre.org/):**
+   - Details adversary tactics, techniques, and procedures (TTPs).
+   - Utilized by offensive security professionals and incident responders.
+
+**2. [OWASP Web Security Testing Guide (WSTG)](https://owasp.org/www-project-web-security-testing-guide/):**
+   - Comprehensive guide focused on web application testing.
+   - Covers high-level phases and specific testing methods for various vulnerabilities.
+
+**3. [NIST SP 800-115](https://csrc.nist.gov/publications/detail/sp/800-115/final):**
+   - Guidelines from the National Institute of Standards and Technology for planning and conducting information security testing.
+
+**4. [Open Source Security Testing Methodology Manual (OSSTMM)](https://www.isecom.org/):**
+   - Document by the Institute for Security and Open Methodologies (ISECOM) outlining repeatable and consistent security testing.
+
+**5. [Penetration Testing Execution Standard (PTES)](http://www.pentest-standard.org/):**
+   - Involves seven distinct phases, including pre-engagement interactions, intelligence gathering, threat modeling, vulnerability analysis, exploitation, post-exploitation, and reporting.
+
+**6. Information Systems Security Assessment Framework (ISSAF):**
```

Comment:
This one is extremely old (https://untrustednetwork.net/files/issaf0.2.1.pdf). Perhaps put a note that is listed for historical purposes? It is definitely outdated.

Reply:
removed in https://github.com/The-Art-of-Hacking/h4cker/pull/187/commits/0a222faa90df6051d9b3528dcd3452c7916d4be5